building-protected-and-paywall.mdx

```diff
@@ -0,0 +1,969 @@
+---
+title: 'How to build paywall pages with Astro, Stripe, Lucia Auth, and Xata'
+description: 'Learn how to build protected and paywall pages using using Astro, Stripe, Lucia Auth, Xata and Vercel.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/gallery-cms-astro-cloudflare.png
+  alt: How to build paywall pages with Stripe, Lucia Auth and Xata
+author: Rishi Raj Jain
+date: 04-17-2024
+published: true
+slug: paywall-astro-stripe-lucia-auth-xata
```

Comment:
I think we can optionally update slug, I do not think they affect google rankings?

Reply:
As I understand it the URL + title help with rankings. I'll update to match the new title.

Reply:
Awesome, thanks!